app/models/agents/data_output_agent.rb

```diff
@@ -95,6 +98,29 @@ def validate_options
       unless options['template'].present? && options['template']['item'].present? && options['template']['item'].is_a?(Hash)
         errors.add(:base, "Please provide template and template.item")
       end
+
+      case options['push_hubs']
+      when nil
```

Comment:
We often treat `''` is nil since it's easier to enter in the GUI, but since push_hubs isn't a default option, I'm okay not doing that.